### DISCUSSION TITLE
Magento 2.4.7 introduces the dependency to symfony/http-foundation ^6.3.4

### DISCUSSION THREAD

**Discussion by creinhardt-on-github** [magewirephp/magewire#123] [Ideas]
Hello Willem,
I'm trying to install magewire along with the latest Magento 2.4.7 beta release from the Mage-OS Nightly Builds repo. Recently, that built got an updated dependency to symfony/http-foundation ^6.3.4:
- magento/magento2-base 2.4.7.1-a20230917+beta1 requires symfony/http-foundation ^6.3.4 -> satisfiable by symfony/http-foundation[v6.3.4].

However, magewire is (still) bound to the ^5.4 release of this symphony component:
- magewirephp/magewire 1.10.7 requires symfony/http-foundation ^5.4 -> satisfiable by symfony/http-foundation[v5.4.0-BETA1, ..., v5.4.28].

Do you have already plans to upgrade this magewire dependency to make it work with the upcoming Magento release?

Thanks, Carsten

**Comment by wpoortman** [comment]
Hi @creinhardt-on-github,

Thanks for starting this discussion. You're always free to create a merge request for it. I'm not really focussed on Magento beta releases, but I think your wishes can easily be fulfilled in the short term.

Lets keep this topic open until that happened.
